### PR TITLE
Adds Pygmalion

### PR DESCRIPTION
**Pull request by EnderMage99**
## About The Pull Request
Adds the abnormality Pygmalion, an aberration of Alriune.
This is the basic info on Pygmalion:
The first Employee that gets a Good work result will gain a mark from now on will be called the “Sculptor”
The “Sculptor” will have a higher work success chance with Pygmalion and will have a higher chance of getting the E.G.O gift
Pygmalion breaches when the “Sculptor” has 50% or less sanity or health."
When Pygmalion breaches, They will teleport to the “Sculptor” and attack any abnormality in their way and any abnormality near the “Sculptor.”
The “Sculptor” will have their prudence reduced to level 2 while Pygmalion is breaching.
If the “Sculptor” goes insane or dies, Pygmalion will become enraged and start attacking everyone near them.

## Why It's Good For The Game

It will hopefully put Alriune back in it place.

### CODE REVIEW

File: code/modules/projectiles/ammunition/ego_ammunition/waw.dm [hunk 1]
```diff
@@ -124,8 +124,13 @@
 	projectile_type = /obj/projectile/ego_bullet/ego_blind_rage
 	pellets = 4
 	variance = 30
-	
+
 /obj/item/ammo_casing/caseless/ego_hookah
 	name = "hookah casing"
 	desc = "A casing from hookah."
 	projectile_type = /obj/projectile/ego_bullet/ego_hookah
+/obj/item/ammo_casing/caseless/ego_bride
```
Comment (Coxswain-Navigator) [inline code/modules/projectiles/ammunition/ego_ammunition/waw.dm:132]: needs a newline to separate it from the previous code

File: code/modules/projectiles/guns/ego_gun/waw.dm [hunk 2]
```diff
@@ -415,6 +415,19 @@
 							TEMPERANCE_ATTRIBUTE = 60,
 							JUSTICE_ATTRIBUTE = 80
 							)
+/obj/item/gun/ego_gun/bride
+	name = "bride"
```
Comment (Coxswain-Navigator) [inline code/modules/projectiles/guns/ego_gun/waw.dm:419]: needs to be changed to "my own bride" or "my own wife" if the headcoder decides to use that instead.

File: code/modules/clothing/suits/ego_gear/waw.dm [hunk 3]
```diff
@@ -342,3 +342,11 @@
 	attribute_requirements = list(
 					PRUDENCE_ATTRIBUTE = 80
 					)
+/obj/item/clothing/suit/armor/ego_gear/bride
+	name = "Bride"
```
Comment (Coxswain-Navigator) [inline code/modules/clothing/suits/ego_gear/waw.dm:346]: ego name needs to be changed

File: code/modules/clothing/suits/ego_gear/waw.dm [hunk 4]
```diff
@@ -342,3 +342,11 @@
...
+	desc = "May your life work, Come back for you."
+	icon_state = "bride"
```
Comment (Kitsunemitsu) [inline code/modules/clothing/suits/ego_gear/waw.dm:348]: Icon state is wife, this duplicates icons.

**Review by Kitsunemitsu — APPROVED** [review]
No issues I see with the code, everything was reported working perfectly fine.

**Pull request merged**